firebase/src/core/domain/models/memo.ts

```diff
@@ -7,7 +7,8 @@ interface MemoContent {
     bold?: boolean;
     italic?: boolean;
     underline?: boolean;
-    codeBlock?: boolean;
+    // Quill uses `kebab-case` format for attributes
+    "code-block"?: boolean;
```

Comment:
Updated to follow the same naming as `quill`

Reply:
We could do as we are doing with the collections (Local & Stored), but all the effort to separate the `Memo` entity into `LocalMemo` and `StoredMemo` doesn't seem to worth it IMO.

Reply:
Exactly, that's why seems to be not worth it to me. The only difference would be this specific `code-block` property.

Reply:
Why the `code-block` would be different from a local to a stored memo?

Reply:
It's just the naming convention. We would have:

```typescript
class LocalMemo {
  "code-block": boolean
}

class StoredMemo {
  codeBlock: boolean
}
```

Reply:
Oh sure, we can assume all of these as the raw quill metadata using their raw representation